contracts/interfaces/IFlashLoanV1Router02.sol

```diff
@@ -0,0 +1,41 @@
+pragma solidity >=0.6.2;
+
+interface IFlashLoanV1Router02 {
+    function factory() external pure returns (address);
+    function WETH() external pure returns (address);
+
+    function addLiquidity(
+        address token,
+        uint amount,
+        address to,
+        uint deadline
+    ) external returns (uint liquidity);
+    function addLiquidityETH(
+        address to,
+        uint deadline
+    ) external payable returns (uint liquidity);
+    function removeLiquidity(
+        address token,
+        uint liquidity,
+        address to,
+        uint deadline
+    ) external returns (uint amount);
+    function removeLiquidityETH(
+        uint liquidity,
+        address to,
+        uint deadline
+    ) external returns (uint amountETH);
+    function removeLiquidityWithPermit(
+        address token,
+        uint liquidity,
+        address to,
+        uint deadline,
+        bool approveMax, uint8 v, bytes32 r, bytes32 s
+    ) external returns (uint amount);
+    function removeLiquidityETHWithPermit(
+        uint liquidity,
+        address to,
+        uint deadline,
+        bool approveMax, uint8 v, bytes32 r, bytes32 s
+    ) external returns (uint amountETH);
+}
```

Comment:
What is the difference between this file with `Router01`?

Reply:
Please check the code structure of `https://github.com/Uniswap/uniswap-v2-periphery/blob/master/contracts/interfaces/IUniswapV2Router02.sol`

Reply:
Done.